packages/langium/src/test/parse-helper.ts

```diff
@@ -0,0 +1,20 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { LangiumDocumentConfiguration } from '../documents/document';
+import { BuildResult } from '../documents/document-builder';
+import { LangiumServices } from '../services';
+
+export function parseHelper(services: LangiumServices): (input: string) => BuildResult {
+    const metaData = services.LanguageMetaData;
+    const documentBuilder = services.documents.DocumentBuilder;
+    return input => {
+        const randomNumber = Math.floor(Math.random() * 10000000) + 1000000;
+        const document = LangiumDocumentConfiguration.create(`file:/${randomNumber}${metaData.fileExtensions[0]}`, metaData.languageId, 0, input);
+        const buildResult = documentBuilder.build(document);
+        return buildResult;
+    };
+}
```

Comment:
We could move this to `langium-test.ts`, or do you see a reason to keep a separate file?